sdk/src/Services/DynamoDBv2/DynamoDBv2.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
The `.sln` file is generated via our SDK generators so any manual changes made on this will not persist. Can you remove the changes to this file?